### PR TITLE
Add periodic_compaction_seconds option to RocksJava

### LINKED ISSUE #8578: Add Java bindings for the `periodic_compaction_seconds` option
`periodic_compaction_seconds` is missing from `Options` and `ColumnFamilyOptions` classes.

### PR DESCRIPTION
**Pull request by zhmaeff**
Fixes #8578

### CODE REVIEW

**Comment by zhmaeff** [comment]
Hi @adamretter could you please review?

File: java/src/main/java/org/rocksdb/ColumnFamilyOptions.java [hunk 1]
```diff
@@ -857,6 +857,17 @@ public long ttl() {
     return ttl(nativeHandle_);
   }
 
+  @Override
+  public ColumnFamilyOptions setPeriodicCompactionSeconds(long periodicCompactionSeconds) {
```
Comment (adamretter) [inline java/src/main/java/org/rocksdb/ColumnFamilyOptions.java:861]: Please mark the parameter as `final`.

File: java/src/main/java/org/rocksdb/MutableColumnFamilyOptions.java [hunk 2]
```diff
@@ -465,5 +466,15 @@ public MutableColumnFamilyOptionsBuilder setTtl(final long ttl) {
     public long ttl() {
       return getLong(CompactionOption.ttl);
     }
+
+    @Override
+    public MutableColumnFamilyOptionsBuilder setPeriodicCompactionSeconds(long periodicCompactionSeconds) {
```
Comment (adamretter) [inline java/src/main/java/org/rocksdb/MutableColumnFamilyOptions.java:471]: Please mark parameter as `final`.

File: java/src/main/java/org/rocksdb/Options.java [hunk 3]
```diff
@@ -1848,6 +1848,17 @@ public long ttl() {
     return ttl(nativeHandle_);
   }
 
+  @Override
+  public Options setPeriodicCompactionSeconds(long periodicCompactionSeconds) {
```
Comment (adamretter) [inline java/src/main/java/org/rocksdb/Options.java:1852]: Please mark parameter as `final`.

**Review by adamretter — CHANGES_REQUESTED** [review]
LGTM just some very minor stuff around `final` parameters.

**Comment by zhmaeff** [comment]
> LGTM just some very minor stuff around `final` parameters.

@adamretter Thank you for a quick review! Fixed!

**Review by adamretter — APPROVED** [review]

**Comment by zhmaeff** [comment]
Hi @mrambacher ! I see some tests are failing https://github.com/facebook/rocksdb/pull/8579/checks?check_run_id=3150162628 , is it related to my code? Is it a blocker for this PR?

**Comment by mrambacher** [comment]
> Hi @mrambacher ! I see some tests are failing https://github.com/facebook/rocksdb/pull/8579/checks?check_run_id=3150162628 , is it related to my code? Is it a blocker for this PR?

Can you update your branch to the latest?  I think everything should be okay after that.

**Comment by zhmaeff** [comment]
2 builds are failing because of 
> "Download snap "cmake" (912) from channel "beta" (received an unexpected http response code (503) when trying to download ...snap"
@mrambacher could you pls rerun them and start internal FB checks?

**Comment by mrambacher** [comment]
Can you please run "make format" and re-push?

**Pull request closed without merging**